ldoc/html/ldoc_ltp.lua

```diff
@@ -68,7 +68,7 @@ return [==[
 # for kind, mods, type in ldoc.kinds() do
 #  if ldoc.allowed_in_contents(type,module) then
 <h2>$(kind)</h2>
-<ul class="$(kind=='Topics' and '' or 'nowrap')">
+<ul class="$(kind=='Topics' and '' or nowrap)">
```

Comment:
This looks incorrect. Are you sure you checked how the Lua string vs. variable parsing is being done here?

Reply:
Yes, this seems to be wrong.